src/Components/Button.js

```diff
@@ -0,0 +1,8 @@
+import React from "react"
+
+const Button = ({label, backgroundColor, onClick, isDisabled}) => {
```

Comment:
isDisable should be an optional prop.

`const Button = ({label, backgroundColor, onClick, isDisabled=false}) => {`

This way the props are optional. If we do not pass the value for these props it will automatically set these default values as props value.